tests/seed/accounts/test_module.py

```
@@ -1,13 +1,12 @@
 import pytest
 
 from mpt_api_client.resources.accounts.modules import AsyncModulesService, Module
-from seed.accounts.module import get_module, refresh_module, seed_module
-from seed.context import Context
+from seed.accounts.module import find_module, seed_module
 
 
 class DummyAsyncIterator:
-    def __init__(self, modules):
-        self._iterator = iter(modules)
+    def __init__(self, items):  # noqa: WPS110
```

Ruff (0.14.8)

Unused `noqa` directive (unknown: `WPS110`)

Remove unused `noqa` directive

(RUF100)